scripts/natural_language_inference/utils.py

```diff
@@ -0,0 +1,25 @@
+import mxnet as mx
+from mxnet import nd
+import gluonnlp as nlp
+import nltk
+
+def fetch_embedding_of_sentence(sentence: str, embedding: nlp.embedding.TokenEmbedding):
+    words = nltk.word_tokenize(sentence.lower())
+    embeds = [embedding[w] for w in words]
+    return nd.stack(*embeds)
+
+def pad_sentences(sentences):
+    length = 0
+    dim = sentences[0].shape[1]
+    for item in sentences:
+        if item.shape[0] > length:
+            length = item.shape[0]
+    ret = []
+    for item in sentences:
+        if length > item.shape[0]:
+            ret.append(
+                    nd.concat(item, nd.zeros((length-item.shape[0], dim)), dim=0))
+        else:
+            ret.append(item)
+    return nd.stack(*ret)
+
```

Comment:
Well, I wish to pad the sequences in a minibatch to the longest length in this batch, rather than padding all sequences into a fixed length. `PadSequence` may not work for this situation.

Reply:
oops. sorry I meant using `batchify`: http://gluon-nlp.mxnet.io/api/data.batchify.html.
E.g. https://github.com/dmlc/gluon-nlp/blob/master/scripts/machine_translation/train_transformer.py#L439